instant-xml/src/lib.rs

```diff
@@ -10,15 +10,47 @@ pub use macros::{FromXml, ToXml};
 pub mod parse;
 
 pub trait ToXml {
-    fn write_xml<W: fmt::Write>(&self, write: &mut W) -> Result<(), Error>;
+    fn write_xml<W: fmt::Write>(
+        &self,
+        write: &mut W,
+        parent_prefixes: Option<&mut BTreeSet<&str>>,
+    ) -> Result<(), Error>;
 
-    fn to_xml(&self) -> Result<String, Error> {
+    fn to_xml(&self, parent_prefixes: Option<&mut BTreeSet<&str>>) -> Result<String, Error> {
```

Comment:
I don't think this needs the `parent_prefixes` argument, that's only necessary for the "internal" `write_xml()` method.

Reply:
It is moved to `Serializer` struct